Content.Client/UserInterface/Screens/SeparatedChatGameScreen.xaml

```diff
@@ -29,7 +29,7 @@
 
             <BoxContainer Orientation="Vertical" HorizontalExpand="True" SeparationOverride="10" Margin="10">
                 <menuBar:GameTopMenuBar Name="TopBar" HorizontalExpand="True" Access="Protected" />
-                <chat:ChatBox VerticalExpand="True" HorizontalExpand="True" Name="Chat" Access="Protected" />
+                <chat:ChatBox VerticalExpand="True" HorizontalExpand="True" Name="Chat" Access="Protected" MinSize="0 0"/>
```

Comment:
The containing PanelContainer already has a minimum width, which is smaller than ChatBox's default.